### ISSUE TITLE
Color order mismatch in Divider component

### ISSUE THREAD

**Issue by Tayebsed93** [Orange-OpenSource/ouds-flutter#216]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

iPhone 14 Pro Max

### OS Version

iOS 18

### App version

v0.4.0

### Describe the issue

The Divider component displays a sequence of colors that does not match the one defined in Figma. The current color order is incorrect, which may lead to visual inconsistency within the application.

### Expected Behavior

The color order in the Divider component should exactly follow the specifications in Figma, ensuring visual consistency with the Orange design system.

- color-border-default
- color border muted
- colord border emphasized
- color border brand primary
- color border on brand primary
- color always black
- color-always-white
- color always on black
- color always on white

**Comment by Tayebsed93** [comment]
📣 New Firebase App Distribution BETA upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: DesignToolbox (BETA) (245)
Version: 0.4.0
Build type: BETA
Build details (GitHub): 245
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @MaximeTonnerre @mccart77 @iassab @TonyAntonyHello (design team)

**Comment by Tayebsed93** [comment]
📣 New TestFlight BETA upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: Design Toolbox Flutter (245)
Version: 0.4.0
Build type: BETA
Build details (GitHub): 245
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @MaximeTonnerre @mccart77 @iassab @TonyAntonyHello (design team)

**Comment by B3nz01d** [comment]
Completed during the sprint 19
